### PR TITLE
Update LNST to use python 3.9

### PR DESCRIPTION
**Pull request by Axonis**
Description:

Updating python version being used in LNST.

Decision to do so, was made based on the fact, that newer poetry installer requires python 3.7 and higher, thus giving us an opportunity to update the LNST version with it as well.

Test:

J:6443576

Reviews: @jtluka @olichtne

### CODE REVIEW

**Comment by Axonis** [comment]
CI testing failed:

The currently activated Python version 3.6.15 is not supported by the project (^3.9).

Seems like we have to fix CI to support python 3.9

**Comment by Axonis** [comment]
I've fixed the CI issue we had with move to a newer poetry installer. Everything seems to be working now, can I get a review @enhaut @olichtne

**Comment by enhaut** [comment]
Looks good to me

**Review by olichtne — APPROVED** [review]

File: .github/workflows/ci-test.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
         sudo apt-get update
         sudo apt-get install podman -y
         sudo systemctl enable --now podman.socket
-        curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3.6 -
+        curl -sSL https://install.python-poetry.org | python3 - --version 1.1.13
```
[the remark this answers is not in the document]
Comment (olichtne) [reply .github/workflows/ci-test.yml:27]: talked to jozef offline and resolved this question - yes this is a newer version of poetry, it shouldn't pose any problems/issues in our deployments though and this was tested.

**Pull request merged**